<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pidecuenta</class>
 <widget class="QDialog" name="pidecuenta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>97</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Cuenta, a partir de ...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Cuenta auxiliar:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buscapushButton">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="keme.qrc">
           <normaloff>:/iconos/graf/viewer.xpm</normaloff>:/iconos/graf/viewer.xpm</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="auxiliarlineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="auxiliarlabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>- -</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelarpushButton">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aceptarpushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>auxiliarlineEdit</tabstop>
  <tabstop>buscapushButton</tabstop>
  <tabstop>cancelarpushButton</tabstop>
  <tabstop>aceptarpushButton</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>pidecuenta</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aceptarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>pidecuenta</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
